cmd/repository/default-pkg-registry_test.go

```diff
@@ -2,21 +2,13 @@ package repository
 
 import (
```

Comment:
We should add one test to cover the load case: a folder with several packages in it